### PR TITLE
[CI:DOCS] Remove broken gate-container docs

### PR DESCRIPTION
**Pull request by cevich**
This container hasn't been used/maintained in ages, remove the related instructions from the contributions doc.

Fixes: #16415

Signed-off-by: Chris Evich <cevich@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -309,49 +309,8 @@ podman build -t gate -f contrib/gate/Dockerfile .
...
-$ git remote update upstream
-$ export EPOCH_TEST_COMMIT=$(git merge-base upstream/master HEAD)
-$ podman run -it --rm -e EPOCH_TEST_COMMIT -v $PWD:/usr/src/libpod:ro \
-    --security-opt label=disable quay.io/libpod/gate:master \
-    validate
-```
-
-### Integration and Other Tests
+### Integration Tests
+>>>>>>> 547a6ff37 ([CI:DOCS] Remove broken gate-container docs)
```
Comment (rhatdan) [inline CONTRIBUTING.md:313]: This looks like a problem?

**Comment by TomSweeneyRedHat** [comment]
LGTM once the rogue line is removed.

**Comment by cevich** [comment]
Oops.  Thanks.  It's fixed.

**Pull request merged**